Investigating NFT Project.ipynb

```diff
@@ -0,0 +1,961 @@
+{
+ "cells":[
+  {
+   "cell_type":"markdown",
+   "source":[
+    "In this notebook is how you can easily start analysing ETH mainnet data by using only open source resources.\n",
+    "\n",
+    "Let's start by installing some usuful libraries for the transactions' analysis.\n",
+    "Here the link to the documentation https:\/\/web3py.readthedocs.io\/en\/stable\/"
+   ],
+   "attachments":{
+    
+   },
+   "metadata":{
+    "datalore":{
+     "type":"MD",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"code",
+   "source":[
+    "!pip install web3==5.25.0\n",
+    "!pip install hexbytes==0.2.2"
+   ],
+   "execution_count":2,
+   "outputs":[
+    
+   ],
+   "metadata":{
+    "datalore":{
+     "type":"CODE",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"code",
+   "source":[
+    "# !pip install  eth-tester"
+   ],
+   "execution_count":3,
+   "outputs":[
+    
+   ],
+   "metadata":{
+    "datalore":{
+     "type":"CODE",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"markdown",
+   "source":[
+    "In order to run the notebook yourself, you may need to create your own account on a node provider website. Here we show you how we do it on Infura: https:\/\/github.com\/shilpachain\/grace_hopper_2022\/blob\/main\/web3_api_demo_for_analytics.ipynb\n",
+    "        "
+   ],
+   "attachments":{
+    
+   },
+   "metadata":{
+    "datalore":{
+     "type":"MD",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"code",
+   "source":[
+    "from web3 import Web3\n",
+    "from web3.auto import w3\n",
+    "from hexbytes import HexBytes\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns"
+   ],
+   "execution_count":4,
+   "outputs":[
+    
+   ],
+   "metadata":{
+    "datalore":{
+     "type":"CODE",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"markdown",
+   "source":[
+    "Put your url here."
+   ],
+   "attachments":{
+    
+   },
+   "metadata":{
+    "datalore":{
+     "type":"MD",
+     "hide_input_from_viewers":false,
+     "hide_output_from_viewers":false
+    }
+   }
+  },
+  {
+   "cell_type":"code",
+   "source":[
+    "my_url = \"https:\/\/mainnet.infura.io\/v3\/df05abee279947a4aaba6515e5bd982a\""
```

Comment:
@giulia-antinori Is this your link? Do you want to keep a placeholder here?

Reply:
Yes you're right. I'll miss it while reviewing I will remove the link.